flocx_market/common/service.py

```diff
@@ -0,0 +1,29 @@
+from oslo_db import options as db_options
+from oslo_log import log
+from oslo_service import service
+
+import flocx_market.conf
+from flocx_market import objects
+from flocx_market import version
+
+CONF = flocx_market.conf.CONF
+
+
+def prepare_service(argv=None, default_config_files=None):
+    argv = [] if argv is None else argv
+    log.register_options(CONF)
+    try:
+        CONF(argv[1:],
+             project='flocx-market',
+             version=version.version_info.release_string(),
+             default_config_files=default_config_files)
+    except SystemExit:
+        raise SystemExit
```

Comment:
I fumbled around with the exceptions for a while. And its probably because I don't understand python exception handling well enough but not having this try catch block there in some capacity wasnt working in the test function and Sahil had to help me to get this working in the way that it is

Reply:
You should remove this `try/except` block.  Can you explain in more detail what sort of problem you were having with the tests?

Reply:
Hmm. I'm not sure what I did differently this time around but now I don't need the try catch. I guess I was looking for the wrong exception on the testers side or something. My bad